### ISSUE TITLE
Design proposal for the menu

### ISSUE THREAD

**Issue by ChristianPauly** [ubports/ubuntu-touch#816]
### Description of the feature
Hey, another simple design proposal: Make the indicators menu a little bit transparent.

The menu is "over" the current app. The user can use this menu without leaving his app. With a transparent effect, it feels more like that he is still in his app and just makes a quick action at the top menu. Also he could still get some information about a change in the current app, for example: I want to look at the calender but in my messenger is a new message. Maybe it is to dark to read the message, but I can see that:

-There has something changed
-The message is very long or very short

For more asthetic: A light blur effect could be a nice eye candy. :-)

### Illustrations
![screenshot20180808_110523265](https://user-images.githubusercontent.com/24619905/43949064-c8d2c176-9c8c-11e8-8425-b88347550cc1.png)
![screenshot20180803_113052171](https://user-images.githubusercontent.com/24619905/43949072-cc423954-9c8c-11e8-9bf6-9ac773fbd853.png)

**Comment by ChristianPauly** [comment]
@PhoenixLandPirate I dont think that this is a good idea. Maybe in a design settings app which fans can install from the openstore but the job of the OS is, to provide a system, that works out of the box with every background. It is not the job of the user to make the system look awesome.

**Comment by PhoenixLandPirate** [comment]
I disagree but I use KDE plasma.

**Comment by PhoenixLandPirate** [comment]
Also perhaps a added blur to the background would be good, this is what Mac, Windows, iOS, KDE and some others do when things are transparent.

**Comment by ChristianPauly** [comment]
Hm yes in general. We could let the user choose what he likes but the slider means he can choose the opacity to make the opacity fit with the image. But this should change automatically. So it would be better to offer a switch, where the user can set the transparency on or off.

**Comment by PhoenixLandPirate** [comment]
I guess that depends what's in the background, these things aren't always accurate to how people view it, I think by default it should be done automatically, but letting people also have access to a slider would be nice if they dont like whats automatic.

**Comment by mateosalta** [comment]
perhaps we could start with the effect the dash is using, then apply that to the current apps screen (like how app switcher takes a sceen for unloaded apps) then we could tint darker.

the argments against is that 
- just trnsparency over text over tex makes things hard to read - thus me should have the blur too
- live blur effect is resource intensive, so we need to do a quick blur on an image to be more effective

**Comment by Fuseteam** [comment]
from my experience of attempting transparency on the terminal setting the opacity around 80% while the text is fully opaque makes the text perfectly readable while providing a nice aesthetic

**Comment by mateosalta** [comment]
> from my experience of attempting transparency on the terminal setting the opacity around 80% while the text is fully opaque makes the text perfectly readable while providing a nice aesthetic
> 
> also I just noticed background of the launcher is actually transparent thinking

it is a fake transparency, taking only the background and blurring, I think we should use the same method that the launcher uses

**Comment by Fuseteam** [comment]
> > from my experience of attempting transparency on the terminal setting the opacity around 80% while the text is fully opaque makes the text perfectly readable while providing a nice aesthetic
> > also I just noticed background of the launcher is actually transparent thinking
> 
> it is a fake transparency, taking only the background and blurring, I think we should use the same method that the launcher uses

I mean I know the drawer is fake transparency, but the launcher too? i noticed I actually see the application that is in the foreground through the launcher

**Comment by mateosalta** [comment]
oh, I sorry I meant the drawer. we should use the drawer method, maybe a bit darker
